.gitignore

```diff
@@ -0,0 +1,205 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
So it doesn't ignore the binplaced locations. Shall we add a line to ignore those too?